test/Autofac.Multitenant.AspNetCore.Test/Autofac.Multitenant.AspNetCore.Test.csproj

```diff
@@ -24,7 +24,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.3.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="5.0.1-develop-00426" />
```

Comment:
I believe this will need to be updated once that PR goes through.

Reply:
Should be good to go. Update it to version `5.0.1-develop-00435`